scribble-lib/scribble/private/tag.rkt

```diff
@@ -0,0 +1,39 @@
+#lang scheme/base
+
+;; It might make sense to make these functions public, but since they weren't originally,
+;; I am going to keep them in the private folder for now.
+
+(require racket/list
+         scribble/core
+         "../tag.rkt")
+
+(provide (all-defined-out))
```

Comment:
Hmm...actually, you might be right. Although I remember needing to require additional things once to use contract out. I'll give it a shot.